packages/perseus/src/widgets/label-image.tsx

```diff
@@ -645,11 +645,15 @@ class LabelImage extends React.Component<LabelImageProps, LabelImageState> {
                 mediaQueries.xsOrSmaller.replace("@media ", ""),
             ).matches;
 
+            // Determine whether the image is wider than it is tall.
+            const isWideImage =
+                this.props.imageWidth / 2 > this.props.imageHeight;
```

Comment:
**question**: Is this a heuristic for deciding if this is essentially a number line question (more than twice as wide as tall)?

Reply:
Kind of! Also looks like our algorithm doesn't account for wide images.

This behavior isn't just to correct for number line images. Any wide images would put pills on the side towards the center (covering each other up) due to the algo only accounting for tall images.